README.md

```diff
@@ -279,7 +279,19 @@ with pub as (
                     case when bool_or(pubtruncate) then 'truncate' else null end
                 ]) act(name_)
         ) w2j_actions,
-        string_agg(cdc.quote_wal2json(prrelid::regclass), ',') w2j_add_tables
+        case
+            -- collect all tables
+            when bool_and(puballtables) then (
+                select
+                    string_agg(cdc.quote_wal2json((schemaname || '.' || tablename)::regclass), ',')
```

Comment:
@olirice we were debugging this earlier and `(schemaname || '.' || tablename)::regclass` doesn't preserve the schema/table name. For example, it converted a table named `CSVTEST5` to `csvtest5` and then Postgres complained that the relation does not exist.

Reply:
At what time did you see the error? i.e.

- Did postgres error on the cast `::regclass`
- Did wal2json error that he table provided to `add-tables` was invalid
or
- Did it error inside `cdc.apply_rls`
- ?

Reply:
@olirice i'll link you the slack thread